### ISSUE TITLE
[2.3.1] Animated SVG Text is placed offscreen

### ISSUE THREAD

**Issue by schiller-manuel** [software-mansion/react-native-reanimated#2900]
## Description

I encountered a bug with `react-native-reanimated` 2.3.1 and `react-native-svg`.

When the app is started, the Text is placed almost entirely offscreen, the left upper screen corner shows a small part of the text.
When I save the file to trigger fast-refresh, the text is rendered at the correct position.

**Demo video**

https://user-images.githubusercontent.com/6340397/147280100-85cc5a42-88bc-439c-a220-5bccb3c54a1d.mp4
### Expected behavior

Text is rendered correctly.

### Actual behavior & steps to reproduce

Text is placed offscreen.

## Snack or minimal code example

https://github.com/schiller-manuel/reanimated-2-playground/tree/svg-bug-2.3.1

## Package versions

| name                                 | version |
| ------------------------------------ | ------- |
| react-native                         |    0.66.4     |
| react-native-reanimated              |    2.3.1     |

## Affected platforms

- [ ] Android (not tested)
- [x] iOS

**Comment by kmagiera** [comment]
This may be fixed by #2580 ? Seems like something that can be caused by the bug that we fix in this PR

**Comment by boodong-rn** [comment]
This is my fake solution for someone who are desperate to solve this issue ASAP.

```jsx
const [isShown, setIsShown] = useState(false);

useEffect(() => {
  const timeoutId = setTimeout(() => {
    setIsShown(true);
  }, 100)
}, [])

// ..

return (
 ...
{isShown && <AnimatedText />}

)

```

**Comment by schiller-manuel** [comment]
> This may be fixed by #2580 ? Seems like something that can be caused by the bug that we fix in this PR

I just tested your PR and it indeed fixes this issue 👍 
Looking forward for a release that contains #2580

https://user-images.githubusercontent.com/6340397/152585744-6c615e7b-eb45-4d18-85b1-b073b5481c2a.mp4
